app/src/components/TokenCard.tsx

```diff
@@ -13,7 +13,7 @@ export const TokenCard = (props: ITokenCardProps): React.ReactElement => {
   return (
     <LinkBase target={`/tokens/${props.token.tokenId}`} isFullWidth={true}>
```

Comment:
Linkbase has prop isFullWidth= true, that is the reason it looks that way on desktop

Reply:
looks what way?